requirements_docs.txt

```diff
@@ -1,7 +1,7 @@
 sphinx>=1.4.3
 sphinxcontrib-napoleon
 sphinx-gallery
-sphinxcontrib-bibtex
+sphinxcontrib-bibtex<2.0.0
```

Comment:
I would say we can remove this pin (and the `>=1.4.3` from `sphinx`), but that would require adding [this](https://github.com/pysal/spaghetti/blob/888047b1efb93a09e30653589cee383ef54d2958/docs/conf.py#L37) to `docs/conf.py` (which seems like doesn't exist?).

Reply:
https://github.com/pysal/pysal/blob/master/docsrc/conf.py ?

Reply:
> https://github.com/pysal/pysal/blob/master/docsrc/conf.py ?

Yeah, I guess I missed that. I'm not sure how the actual PySAL docs get built, or if they are simply built at the submodule-level.